templates/node/src/index.ts.twig

```diff
@@ -7,6 +7,7 @@ export type { QueryTypes, QueryTypesList } from './query';
 export { Permission } from './permission';
 export { Role } from './role';
 export { ID } from './id';
+export { InputFile } from './inputFile';
```

Comment:
It's now on a seperate entry point. So instead you do 

```
import { InputFile } from 'node-appwrite/file'
```

If we put it on this entrypoint edge runtimes will break as inputfile depends on node fs module

Reply:
Is that a separate package? I wasn't getting any suggestions on what to import:

<img width="662" alt="image" src="https://github.com/appwrite/sdk-generator/assets/1477010/fed89900-64c9-4b48-98d4-548e8419917d">

That import also didn't work for me:

<img width="903" alt="image" src="https://github.com/appwrite/sdk-generator/assets/1477010/375f8297-c8c6-4156-a106-cfb89ab1dae2">

Where is this documented?

Reply:
https://github.com/appwrite/sdk-generator/blob/3ade92cbc7f6498c024314199f31237cfe13c80d/templates/node/package.json.twig#L23

Should be defined here in the package.json

Not sure why you don't have the types

Reply:
@loks0n, and it works for you? Can you try on the https://github.com/appwrite/playground-for-node repo?

Reply:
FYI, if I just put /, I get this as autocomplete:

<img width="755" alt="image" src="https://github.com/appwrite/sdk-generator/assets/1477010/eca1b093-0ed9-4407-84f9-c17c452648f8">

Reply:
Works for me with CJS and ESM. Are you 100% sure you have the rc version installed?
<img width="868" alt="Screenshot 2024-05-10 at 10 36 45" src="https://github.com/appwrite/sdk-generator/assets/22452787/408858f3-206c-47fc-a1bb-e98398391438">

Reply:
Ahh looks like I had a bad tsconfig.json file. It had `module` set to `commonjs` and maybe something else that caused problems. I created a fresh project and was able to resolve the dependency fine 🎉